project_national/housing_characteristics/HVAC System Single Speed AC Airflow.tsv

```diff
@@ -22,4 +28,4 @@ Room AC, EER 10.7	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.0941834
 Room AC, EER 12.0	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.0346726
 Shared Cooling	No	0	0	0	0	0	0	0	0	0	0	0	0	1	0.0185925
 # Created by: manually created
-# Source: Winkler et al. 'Impact of installation faults in air conditioners and heat pumps in single-family homes on US energy usage' 2020
\ No newline at end of file
+# Source: Winkler et al. 'Impact of installation faults in air conditioners and heat pumps in single-family homes on US energy usage' 2020
```

Comment:
probably should update docs here to note that we changed stuff

Reply:
Done.